src/bin/config-store.rs

```diff
@@ -1,10 +1,11 @@
 use anyhow::anyhow;
 use clap::{Parser, Subcommand};
 use derive_more::Display;
+use gosub_engine::config;
 use gosub_engine::config::settings::Setting;
-use gosub_engine::config::storage::json_storage::JsonStorageAdapter;
-use gosub_engine::config::storage::sqlite_storage::SqliteStorageAdapter;
-use gosub_engine::config::{ConfigStore, Store};
+use gosub_engine::config::storage::json::JsonStorageAdapter;
+use gosub_engine::config::storage::sqlite::SqliteStorageAdapter;
+use gosub_engine::config::StorageAdapter;
```

Comment:
The name "store" was a bit ambiguous. They are actually storage adapters since we already have a config store.

Reply:
@Sharktheone Can you guve an example on what you mean?

Reply:
I mean, doing this in `storage.rs`

```rust
mod json;
mod memory;
mod sqlite;

pub use json::*;
pub use memory::*;
pub use sqlite::*;
```

Then you can do this
```rust
use gosub_engine::config::storage::*
```

and still use `JsonStorageAdapter` and `SqliteStorageAdapter`. This makes things cleaner. Most of the time when you import one of the storage atapters, you most likely need all.